src/spirc.rs

```diff
@@ -170,6 +172,16 @@ impl<D: SpircDelegate> SpircManager<D> {
             protocol::spirc::MessageType::kMessageTypePause => {
                 self.delegate.pause();
             }
+            protocol::spirc::MessageType::kMessageTypeNext => {
+            	self.index = (self.index + 1) % self.tracks.len() as u32;
+                let track = self.tracks[self.index as usize];
```

Comment:
Hahaha ok that look was a bit too quick,  i was working with matlab all day yesterday which has start index 1..  I simply miscounted the list thinking there was a track inbetween. Things make sense again,  thanks ;)
Will try adding functionality aap.